internal/deb/extract.go

```diff
@@ -24,8 +24,9 @@ type ExtractOptions struct {
 	Package   string
 	TargetDir string
 	Extract   map[string][]ExtractInfo
-	Globbed   map[string][]string
-	Create    func(options *fsutil.CreateOptions) error
+	// Create creates a fs entry given the options. If the entry is extracted from
+	// the tarball, extractInfo should not be empty, else nil can be passed.
```

Comment:
The comment seems incorrect. Given the code right below, extractInfo is nil in cases where we do have the content coming from the tarball, explicitly so (there's literally an independent entry from the parent directory), but it's not explicitly listed in Extract above (parent directories).

So this should look along the lines of:
```
// Create can optionally be set to control the creation of extracted entries.
// extractInfo is set to the matching entry in Extract, and is nil in cases where
// the created entry is implicit and unlisted (for example, parent directories).
```

Reply:
I agree, and the example you sent is great! I have put that verbatim in the code.